test-utils/testlib/src/node/process_node.rs

```diff
@@ -138,17 +138,19 @@ impl ProcessNode {
     pub fn get_start_node_command(&self) -> Command {
         if let Err(_) = std::env::var("NIGHTLY_RUNNER") {
             let mut command = Command::new("cargo");
-            command.args(&[
-                "run",
-                "-p",
-                "neard",
-                "--bin",
-                "neard",
-                "--",
-                "--home",
-                &self.work_dir,
-                "run",
-            ]);
+            let mut args = vec!["run", "-p", "neard"];
+            #[cfg(feature = "nightly_protocol")]
+            {
+                args.push("--features");
+                args.push("nightly_protocol");
+            }
+            #[cfg(feature = "nightly_protocol_features")]
+            {
+                args.push("--features");
+                args.push("nightly_protocol_features");
+            }
+            args.append(&mut vec!["--bin", "neard", "--", "--home", &self.work_dir, "run"]);
+            command.args(&args[..]);
             command
```

Comment:
```suggestion
            command.args(&["run", "-p", "neard"]);
            #[cfg(feature = "nightly_protocol")]
            command.args(&["--features", "nightly_protocol"]);
            #[cfg(feature = "nightly_protocol_features")]
            command.args(&["--features", "nightly_protocol_features"]);
            command.args(&["--bin", "neard", "--", "--home", &self.work_dir, "run"]);
            command
```

This is a more direct way to express this, you can call `.args` or `.args` on a `Command` several times.